.github/workflows/subsumption-bundle-verifier.yml

```
@@ -15,6 +15,10 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Use Node
+      - name: Setup pnpm
+        uses: pnpm/action-setup@c5ba7f7862a0f64c1b1a05fbac13e0b8e86ba08c # v4
+        with:
+          version: 9.12.0
         uses: actions/setup-node@v4
```

YAMLlint (1.38.0)

[error] duplication of key "uses" in mapping

(key-duplicates)